docs/marketing/detailed-description.md

```diff
@@ -0,0 +1,18 @@
+The KOMOJU extension provides an easy way to accept Japanese payments for your Magento store. Credit Card payments only make up a small percentage of online payments in Japan, with KOMOJU you can accept payments in multiple ways, allowing your customers to pay in the way that suits them best.
+
+Note: Currently the extension only allows payments for stores with JPY as the base currency.
+
+## Account & Pricing
+
+A KOMOJU account is required to use this extension. If you do not have a KOMOJU account yet you will need to [sign up here](https://komoju.com/) for one before using the extension. See [this link](https://komoju.com/#pricing) for pricing details.
+
+## Security
+
+- PCI Compliant
+- No Credit Card details stored in Magento
+
+## Features
+
+- Allow your customers to pay with the methods you prefer best: Credit Card or Konbini
```

Comment:
Can we modify this description after release if we decide to add more payment methods?

Reply:
Yep! It will require us to submit it for review again, but since we would have to make changes to the code to support additional payment types we would have to get it re-reviewed anyway.

[Source](https://devdocs.magento.com/marketplace/sellers/extension-update-information.html#update-marketing-information)